src/components/Spinner/Spinner.st.css

```diff
@@ -0,0 +1,60 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:vars {
+    DefaultPathColor: color(color-5);
+}
+
+:vars {
+    PathColor: --overridable;
+}
+
+:vars {
+    UsedPathColor: color(fallback(value(PathColor), value(DefaultPathColor)));
+}
+
+.root {
+    -st-states: mobile, centered, static;
+
+    animation: rotate 2s linear infinite;
+    z-index: 2;
+}
+
+.circle {
+    stroke: value(UsedPathColor);
+    animation: dash 1.5s ease-in-out infinite;
+}
+
+.root:centered {
+    position: absolute;
+    left: 50%;
```

Comment:
is `left: 50%` enough?
Maybe we need to add `transform: translateX(-50%);`?

Reply:
btw, do we also want to center it vertically?

Reply:
Fixed, now top and left are setted inside jsx, in accordance to diameter prop